contrib/pyudmi/src/udmi/config.py

```diff
@@ -0,0 +1,57 @@
+"""
+{
```

Comment:
what are these?  They seem redundant with the schema definitions in the UDMI/schema directory -- redundancy is bad (much harder to maintain), so I'm trying to understand what they are... ?

Reply:
The schema json strings in these files are documentation only. The python models are oo way to create udmi objects that are validated on creation

Reply:
sure - i'll take take these strings out